### ISSUE TITLE
Loading game: OK button stays enabled when last savegame was deleted

### ISSUE THREAD

**Issue by bunnybot** [widelands/widelands#7000]
[<img align='left' src='https://codeberg.org/avatars/368c0c987e1160f8f3180310d5bce945' alt='JonasMoerschbach' height='48' />](https://codeberg.org/JonasMoerschbach)*[Mirrored from Codeberg](https://codeberg.org/wl/widelands/issues/5359)*
*Created on Sat Apr 04 15:41:47 CEST 2026 by [**JonasMoerschbach**](https://codeberg.org/JonasMoerschbach)*

---

### Describe the Bug

The "Ok"-button's behavior in Load UI is confusing. It should be disabled when no savegame is selected

### To Reproduce

1. Have at least one single player savegame
2. In main menu, select "Single Player" and then "Load game"
3. Delete all save games in the table
4. Watch the buttons: Delete button is correctly disabled when no savegame is available, but Ok button stays enabled. Clicking it does nothing

### Expected Behavior

4. the Ok-Button should be disabled when no savegame is selected

### Operating System

Linux Mint (but for sure OS independent)

### Widelands Version

"1.4"
